MIGRATION.md

```diff
@@ -23,7 +23,13 @@ There are several ways to create a client connection in `cloudant-java-sdk`:
 1. There is no built-in pagination support for views. Examples coming soon.
 1. Replay interceptors are replaced by the [automatic retries](https://github.com/IBM/ibm-cloud-sdk-common/#automatic-retries) feature for failed requests.
 1. Error handling is not transferable from `java-cloudant` to `cloudant-java-sdk`. For more information go to the [Error handling section](https://cloud.ibm.com/apidocs/cloudant?code=java#error-handling) in our API docs.
-1. Custom HTTP client configurations in `java-cloudant` are not transferable to `cloudant-java-sdk`. For more information go to the [Configuring the HTTP client section(https://githubcom/IBM/ibm-cloud-sdk-common/#configuring-the-http-client) in the IBM Cloud SDK Common README.
+1. Custom HTTP client configurations in `java-cloudant` are not transferable to 
+   `cloudant-java-sdk`. For more information go to the [Configuring the HTTP client section]
+   (https://github.com/IBM/ibm-cloud-sdk-common/#configuring-the-http-client) in the IBM Cloud SDK Common README.
+1. Authentication errors turn out at the time of instantiation of a client, while errors
```

Comment:
I think we should split this into two items.
Wdyt about these changes:
```suggestion
1. Authentication errors occur during service instantiation.  For example, the code `Cloudant service = Cloudant.newInstance("EXAMPLE");` will fail with `Authentication information was not properly configured.` if required environment variables prefixed with `EXAMPLE` are not set.
1. Server errors occur when running a request against the service.  We suggest to
   check server errors with [`getServerInformation`](https://cloud.ibm.com/apidocs/cloudant?
    code=java#getserverinformation) which is the new alternative of `metaInformation()`.
```

Reply:
Done in a91600a